supervision/detection/utils.py

```diff
@@ -596,6 +596,44 @@ def move_boxes(
     return xyxy + np.hstack([offset, offset])
 
 
+def move_obb_boxes(
+    xyxyxyxy: npt.NDArray[np.float64], offset: npt.NDArray[np.int32]
+) -> npt.NDArray[np.float64]:
+    """
+    Parameters:
+        xyxyxyxy (npt.NDArray[np.float64]): An array of shape `(n, 4, 2)` containing the
+        bounding boxes coordinates in format
+        `n[[x1, y1], [x2, y2],[x3, y3], [x4, y4]]`
+        offset (np.array): An array of shape `(2,)` containing offset values
+        in format is `[dx, dy]`.
+
+    Returns:
+        npt.NDArray[np.float64]: Repositioned oriented bounding boxes.
+
+    Examples:
+        ```python
+        import numpy as np
+        import supervision as sv
+
+        xyxyxyxy: np.array([[[10., 10.],
+                            [20., 20.],
+                            [30., 30.],
+                            [40., 40.]]])
+        offset = np.array([5, 10])
+
+        sv.move_obb_boxes(xyss, offset=offset)
+        #[[[15., 20.],
+        #[25., 30.],
+        #[35., 40.],
+        #[45., 50.]]]
+        ```
+    """
+    return [
```

Comment:
I just wanted to let you know that you don't need to do it like this. Numpy supports broadcasting. Broadcasting allows NumPy to perform element-wise operations on arrays of different shapes by automatically expanding the smaller array to match the shape of the larger one.

You can simply `xyxyxyxy + offset`.

You can learn more from this colab: https://colab.research.google.com/drive/10FVlOMQaStrg7yUo-_x4l_NEBafHzUqI?usp=sharing